pkg/model/project.go

```diff
@@ -451,21 +451,20 @@ func (p *Project) DeleteUserGroup(sso string) error {
 	return fmt.Errorf("%s user group does not exist", sso)
 }
 
-// GetAllRBACRoles returns user groups and built-in roles.
-func (p *Project) GetAllRBACRoles() []*ProjectRBACRole {
+// SetBuiltinRBACRoles sets built-in roles.
+func (p *Project) SetBuiltinRBACRoles() *Project {
 	builtin := []*ProjectRBACRole{
 		builtinAdminRBACRole,
 		builtinEditorRBACRole,
 		builtinViewerRBACRole,
 	}
-
 	all := make([]*ProjectRBACRole, 0, len(p.RbacRoles)+len(builtin))
 	// Set built-in rbac role.
 	all = append(all, builtin...)
 	// Set custom rbac role.
 	all = append(all, p.RbacRoles...)
-
-	return all
+	p.RbacRoles = all
+	return p
```

Comment:
we may not need to return itself here

Reply:
This is because I wanna use method-chain pattern but this is just my preference.
If you guys feel it's weird, let me fix this.

Reply:
I understand your thought, but currently we have [SetStaticAdmin](https://github.com/pipe-cd/pipecd/blob/0ce66edebbd26b65e65444a20ffb393ea802df14/pkg/model/project.go#L149) and SetUpdatedAt which is setter as well, so I guess we should keep it unify with others.

Reply:
OK